quixstreams/sources/community/mysql_cdc/mysql_cdc.py

```diff
@@ -0,0 +1,251 @@
+import logging
+import time
+from typing import Optional
+
+from quixstreams.sources.base import StatefulSource
+
+from .mysql_helper import MySqlHelper
+
+__all__ = ("MySqlCdcSource",)
+
+logger = logging.getLogger(__name__)
+
+
+class MySqlCdcSource(StatefulSource):
+    def __init__(
+        self,
+        host: str,
+        port: int,
+        user: str,
+        password: str,
+        database: str,
+        table: str,
+        state_dir: str = "state",
+        initial_snapshot: bool = False,
+        snapshot_host: Optional[str] = None,
+        snapshot_batch_size: int = 1000,
+        force_snapshot: bool = False,
+        name: str = "mysql-cdc-source",
+    ):
+        super().__init__(name=name)
+
+        self._database = database
+        self._table = table
+        self._table_name = f"{self._database}.{self._table}"
+        self.wait_interval = 0.1
+        self._helper = MySqlHelper(
+            **{
+                "host": host,
+                "port": port,
+                "user": user,
+                "password": password,
+                "database": database,
+                "table": table,
+                "snapshot_host": snapshot_host or host,
+                "state_dir": state_dir,
+            }
```

Comment:
I think it doesn't need to be a dict
```suggestion
            host=host,
            port=port,
            user=user,
            password=password,
            database=database,
            table=table,
            snapshot_host=snapshot_host or host,
            state_dir=state_dir,
```